external/fv3fit/fv3fit/reservoir/config.py

```diff
@@ -102,10 +103,19 @@ def __post_init__(self):
                 f"Output variables {self.output_variables} must be a subset of "
                 f"input variables {self.input_variables}."
             )
+        if (
+            self.hybrid_variables is not None
+            and len(set(self.hybrid_variables).intersection(self.input_variables)) > 0
```

Comment:
nit: I tend to take these long sequences and use a well-named intermediate variables for readability